content-service/src/contentReplicator.js

```diff
@@ -68,10 +68,28 @@ class ContentReplicator {
     return highestVal
   }
 
+  async monitorDiscoveryProviderConnection () {
+    let currentDiscProv = this.audiusLibs.discoveryProvider.discoveryProviderEndpoint
+    let retries = 5
+    while (!currentDiscProv && retries > 0) {
+      logger.info(`Reselecting discovery provider`)
+      await this.audiusLibs.init()
+      currentDiscProv = this.audiusLibs.discoveryProvider.discoveryProviderEndpoint
+      retries--
```

Comment:
@hareeshnagaraj do you want to put a timeout here for like a few seconds to separate this? otherwise this will run 5 at once and maybe not give it enough time to initialize?